src/actions/commentsActions.js

```diff
@@ -0,0 +1,62 @@
+import {
+  CREATE_COMMENTS,
+  CREATE_COMMENTS_ERRORS,
+  FETCH_COMMENTS,
+  FETCH_COMMENTS_ERRORS
+} from "./types";
+import axios from "axios";
+import { toast } from "react-toastify";
+
+export const articleComments = (data, slug) => async dispatch => {
+  return await axios
+    .post(
+      `https://ah-backend-prime-staging.herokuapp.com/api/v1/articles/${slug}/comments/0/`,
+      data,
+      {
+        headers: {
+          Authorization: "Bearer " + sessionStorage.token
+        }
+      }
+    )
+    .then(res => {
+      dispatch({
+        type: CREATE_COMMENTS,
+        payload: res.data
+      });
+      toast.dismiss();
+      toast.success("thanks for your view", {
+        hideProgressBar: false,
+        autoClose: 3000
+      });
+    })
+    .catch(errors => {
+      console.log(errors.response.data);
```

Comment:
Hey @anyatibrian , great work. Kindly remove the console.logs in your code.

Reply:
done